OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -731,7 +731,7 @@ def log_and_exit(hutil, operation, exit_code = 1, message = ''):
     if hutil is not None:
         hutil.do_exit(exit_code, operation, exit_status, str(exit_code), message)
     else:
-        update_status_file(operation, exit_code, exit_status, message)
+        update_status_file(operation, str(exit_code), exit_status, message)
```

Comment:
This seems a regression from the previous commit. Can you update the description of this commit to state so?
Also please send them in separate PR, it is easy for the reviewer miss something if there are more than one unrelated commits in a single PR.